.isort.cfg

```diff
@@ -1,6 +1,9 @@
 [settings]
 line_length=110
-known_third_party=twisted,migrate,sqlalchemy,ldap3,klein,future,requests,MySQLdb,zope,coverage,autobahn,pkg_resources,jinja2,mock,dateutil,sphinx,setuptools
+known_future_library=future
+known_twisted=twisted,zope,autobahn,klein
```

Comment:
aha, thanks for the reminder.

then order of 3rd party and twisted + mock after 3rd party are the only things standing between me and +2.